src/web.rs

```diff
@@ -74,6 +72,21 @@ impl WebImpl {
         })
     }
 
+    fn resolve_ctx<T: JsCast>(
```

Comment:
A comment explaining what this function does would be nice

Reply:
This function is useless right now, will re-introduce it in #103.
Removed it for now.